### ISSUE TITLE
MultipleClassFileToPsr4ClassesRector - Skipping non-namespaced classes

### ISSUE THREAD

**Issue by JanMikes** [rectorphp/rector#1422]
Hi!

In our project, we have few classes that are not namespaced and are in same file -> I wanted to split them, but found out, they are skipped, I am sure it is because of this condition:

```php
// process only namespaced file
 if ($namespaceNodes === []) {
    return true;
}
```

What is reason these are skipped? Is it because of current implementation, because it is working with namespace nodes and it is missing?

If it is only reason, i could try to reimplement so it is possible to refactor non-namespaced classes as well.

**Comment by TomasVotruba** [comment]
> Is it because of current implementation, because it is working with namespace nodes and it is missing?

Indeed. 

> If it is only reason, i could try to reimplement so it is possible to refactor non-namespaced classes as well.

Go for it :+1:

**Comment by TomasVotruba** [comment]
Closed by #1460
